packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -2926,6 +2931,9 @@ void import_and_extract_views(
 
     } else if (prototypeImporter.is_null()) {
       // No prototype importer --- count the remotes the hard way
+#ifdef HAVE_TPETRA_MMM_TIMINGS
+      TimeMonitor MM2 = *TimeMonitor::getNewTimer(prefix_mmm + std::string("MMM I&X RemoteMap-Mode2"));
```

Comment:
I realized this compiles only because `TimeMonitor(Timer&)` is not marked `explicit`.  At least TimeMonitor isn't being handled by RCP here ;-) .